### ISSUE TITLE
Weird display when isHorizontalScrollEnabled=false

### ISSUE THREAD

**Issue by JamalMulla** [patrykandpatryk/vico#90]
When `isHorizontalScrollEnabled=false`, the chart is displayed as a single vertical line. It only displays properly after the chart is touched. This doesn't happen if the flag isn't set. Is there any other way to show the whole data without scrolling?

I'm using the jetpack compose Chart with lineChart()

**Comment by patrykmichalik** [comment]
Hello! Thanks for the bug report. We can’t seem to reproduce this issue on our end. Would you be able to provide a minimal reproducible example and a screenshot or screen recording of the bug? Cheers!

**Comment by JamalMulla** [comment]
I've attached a video of the bug along with a minimal example that reproduces it in JC. I'm using v1.0.0. In this minimal example, I actually need to try and zoom for the graph to show up.

```
@Composable
fun ComposeLineChart(
    color: Color,
) {

    val stuff = entryModelOf(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,)

    ProvideChartStyle(chartStyle = m3ChartStyle(entityColors = listOf(color))) {

        Chart(
            chart = lineChart(),
            model = stuff,
            startAxis = startAxis(),
            bottomAxis = bottomAxis(),
            isHorizontalScrollEnabled = false,
        )
    }
}
```
[Video](https://imgur.com/a/YOZbeXk)

**Comment by patrykmichalik** [comment]
Thanks, @JamalMulla! I’ve managed to reproduce the issue and we’re looking into it. Apologies for the inconvenience.

**Comment by patrykmichalik** [comment]
Hello again! This issue is resolved in Vico 1.1.0, which should be available within half an hour. Thanks for the bug report!
